scripts/run_example

```diff
@@ -110,37 +110,37 @@ fi
 readonly CLIENT_ARGS=("${@-}")  # Choose client args provided after '--'.
 
 # Run the application clients.
-for client_language in $client_languages
+for client_variant in $client_variants
 do
-  case "${client_language}" in
-    rust)
+  case "${client_variant}" in
+    cargo)
       rust_client_manifest="${SCRIPTS_DIR}/../examples/${EXAMPLE}/client/rust/Cargo.toml"
       if [[ -f "${rust_client_manifest}" ]]; then
           cargo run --release --target=x86_64-unknown-linux-musl --manifest-path="${rust_client_manifest}" -- \
             --root-tls-certificate="${SCRIPTS_DIR}/../examples/certs/local/ca.pem" \
             "${CLIENT_ARGS[@]-}"
       else
-        echo "The ${EXAMPLE} example does not contain a ${client_language} client. Skipping this client."
+        echo "The ${EXAMPLE} example does not contain a ${client_variant} client. Skipping this client."
       fi
       ;;
-    cpp)
-      cpp_client="./bazel-client-bin/examples/${EXAMPLE}/client/client"
-      if [[ -f "${cpp_client}" ]]; then
-          "${cpp_client}" "${TLS_ARGS[@]}" "${ADDITIONAL_ARGS[@]-}" "${CLIENT_ARGS[@]-}"
+    bazel)
+      build_file="${SCRIPTS_DIR}/../examples/${EXAMPLE}/client/BUILD"
+      if [[ -f "${build_file}" ]]; then
+          "${SCRIPTS_DIR}/../bazel-client-bin/examples/${EXAMPLE}/client/client" "${TLS_ARGS[@]}" "${ADDITIONAL_ARGS[@]-}" "${CLIENT_ARGS[@]-}"
```

Comment:
Probably outside of scope of this review, but we should probably use `bazel run ....` and let it figure out paths instead of manually pointing to the file.

Reply:
Agreed. Though given I heard we also want to migrate away from bazel, I'm not sure if we want to invest more effort into our bazel code rn. :)